tools/mc-java/src/main/java/io/spine/tools/mc/java/codegen/Names.java

```diff
@@ -24,24 +24,37 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-package io.spine.tools.mc.java.protoc.method.given;
+package io.spine.tools.mc.java.codegen;
 
-import io.spine.tools.protoc.Classpath;
-import io.spine.tools.mc.java.protoc.ExternalClassLoader;
-import io.spine.tools.protoc.NestedClassFactory;
+import io.spine.annotation.Internal;
+import io.spine.tools.protoc.JavaClassName;
 
 /**
- * A test wrapper for the {@link ExternalClassLoader}.
+ * A test factory of {@code JavaClassName}s.
  */
-public final class TestClassLoader {
+@Internal
+public final class Names {
 
-    private static final ExternalClassLoader<NestedClassFactory> INSTANCE =
-            new ExternalClassLoader<>(Classpath.getDefaultInstance(), NestedClassFactory.class);
+    /**
+     * Prevents the utility class instantiation.
+     */
+    private Names() {
+    }
 
-    private TestClassLoader() {
+    /**
+     * Obtains a {@code JavaClassName} with the given value.
+     */
+    public static JavaClassName className(String name) {
```

Comment:
Even considering this type is `@Internal`, I would check the arguments before working with them. It's here and in the method below.